frontend/package.json

```diff
@@ -12,6 +12,9 @@
     "@hookform/error-message": "^0.0.5",
     "@trussworks/react-uswds": "1.11.0",
     "@use-it/interval": "^1.0.0",
+    "draft-js": "^0.11.7",
+    "draftjs-to-html": "^0.9.1",
+    "html-to-draftjs": "^1.5.0",
     "http-proxy-middleware": "^1.0.5",
```

Comment:
`draft-js` is the core editor that our `wysiwyg` editor is built upon. The other two deps are to transform the editor data to/from html.